electron/src/main.ts

```diff
@@ -184,6 +185,10 @@ const showMainWindow = (mainWindowState: WindowStateKeeper.State) => {
     webappURL += `&enableLogging=@wireapp/*`;
   }
 
+  if (customProtocolHandler.hashLocation) {
+    webappURL += `#${customProtocolHandler.hashLocation}`;
```

Comment:
What do you think about using Node's [`url`](https://nodejs.org/api/url.html) module to build the URL? Constructing the URL is getting a bit fiddly and I am also wondering if it works with `encodeURIComponent()` in line 196.

Reply:
It's probably a good idea, indeed, to use node's url builder :+1: 
There should be no problem with the `encodeUIRComponent`, we still need that as it is passed as a url parameter

Reply:
@ffflorian I will check if we can get rid off the `file-url` package which is being used here.